bin/fin

```diff
@@ -2181,14 +2275,14 @@ docker_machine_create ()
 
 	# Use a local boot2docker.iso copy when available
 	ISO_FILE=${ISO_FILE:-boot2docker.iso}
-	if [[ -f "$ISO_FILE" ]]; then
-		echo "Found $ISO_FILE. Using it..."
-		URL_BOOT2DOCKER="$ISO_FILE"
+	if [[ -f "${ISO_FILE}" ]]; then
+		echo "Found ${ISO_FILE}. Using it..."
```

Comment:
I thought you wanted to get rid of it because it's not working on WSL

Reply:
This works fine on WSL as long as install/update is done within a Windows folder. I'll add a note about this in the portable installation docs.